src/components/user/AuthorProfile.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CardBody,
+  CardText,
+  CardFooter,
+  Col,
+  Row,
+} from 'reactstrap';
+import { fetchAuthorProfile } from '../../store/actions/profileActions';
+import FollowUnfollow from './FollowUnfollow';
+import UserStats from './UserStats';
+
+
+export class AuthorProfile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      authorProfile: {},
+    };
+  }
+
+  componentDidMount = () => {
+    const { match } = this.props;
+    const { params } = match;
+    const { username } = params;
+    const { getAuthorProfile } = this.props;
```

Comment:
@BrianSerem, destructure all props on one line.

Reply:
I have destructured both getAuthorProfile and match on the same line.